weaviate/collection/classes/orm.py

```diff
@@ -33,7 +33,7 @@
 
 @dataclass
 class CrossReference:
-    ref_type: Union[Type, str]
+    ref_type: Union[type, str]
 
```

Comment:
why this change? Also 3 lines down it still uses "type"

Reply:
Potentially a relic mypy change, will redo